### PR TITLE
deprecate nn.keras

### PR DESCRIPTION
**Pull request by Le-Zheng**
## What changes were proposed in this pull request?

deprecate nn.keras
1. Python: add deprecate notes for layers
2. Scala: deprecate layers/api which is not extended by ZOO

## How was this patch tested?

will be tested on Jenkins

## Related links or issues (optional)
To fix https://github.com/intel-analytics/BigDL/issues/3005

### CODE REVIEW

**Comment by Le-Zheng** [comment]
passed Jenkins test: http://10.239.47.210:18080/job/BigDL-PRVN-lite/1149/

**Comment by Le-Zheng** [comment]
There is no cleverer way but to deprecate each class as `nn.kreas` is a namespace.

File: pyspark/bigdl/nn/keras/topology.py [hunk 1]
```diff
@@ -200,6 +204,9 @@ class Sequential(KerasModel):
 
     >>> sequential = Sequential(name="seq1")
     creating: createKerasSequential
+
+    .. note:: `bigdl.nn.keras` is deprecated in 0.11.
+    This will be removed in future releases.
     """
     def __init__(self, jvalue=None, **kwargs):
```
Comment (hkvision) [inline pyspark/bigdl/nn/keras/topology.py:211]: Do we need to add a warning in init?

Comment (jason-dai) [reply pyspark/bigdl/nn/keras/topology.py:211]: In general no - Analytics Zoo will also used these classes, yes?

Comment (hkvision) [reply pyspark/bigdl/nn/keras/topology.py:211]: No. For the Python side, Analytics Zoo's all keras layers are newly wrapped and have no relationship with bigdl's keras layers. The dependencies only exist on Scala side.

Comment (Le-Zheng) [reply pyspark/bigdl/nn/keras/topology.py:212]: Done. On the scala side, only deprecate Sequential, Model, and Input class. On the python side, add a warning for the Sequential and Model class.

**Pull request merged**